src/client/pages/ResetPassword.tsx

```diff
@@ -26,45 +28,45 @@ export const ResetPassword = ({
   buttonText,
   queryString = '',
   formActionOverride,
+  inputLabel,
```

Comment:
small one, but could be worth being specific around the name `inputLabel` as it seems to be mainly used as an `emailInputLabel` across the pages 🤔